Header header
float64[3] position
float64[3] velocity
float64[3] acceleration
float64[4] attitude_quaternion
float64[4] angular_velocity